boxoffice/static/css/app.css

```diff
@@ -665,6 +665,14 @@ input.icon-placeholder {
   overflow-x: inherit;
 }
 
+@media (min-width: 768px) {
+  h1.boxoffice-listing {
+    margin-left: 3em;
+    border-bottom: 2px solid;
```

Comment:
Is border bottom required?

Reply:
It makes for a nicer separation between the title and the widget on widget. But, of course, it's a subjective assessment.